### PR TITLE
fix azurerm_synapse_workspace_key doc example

### PR DESCRIPTION
**Pull request by ms-henglu**
To address issue: https://github.com/hashicorp/terraform-provider-azurerm/issues/14209

### CODE REVIEW

File: website/docs/r/synapse_workspace_keys.html.markdown [hunk 1]
```diff
@@ -107,7 +109,7 @@ resource "azurerm_synapse_workspace_key" "example" {
   customer_managed_key_versionless_id = azurerm_key_vault_key.example.versionless_id
   synapse_workspace_id                = azurerm_synapse_workspace.example.id
   active                              = true
-  key_name                            = "enckey"
+  cusomter_managed_key_name           = "enckey"
```
Comment (stephybun) [inline website/docs/r/synapse_workspace_keys.html.markdown:112]: This field is deprecated so could we use `customer_managed_key_name` instead?

**Review by stephybun — COMMENTED** [review]
Thanks for fixing this @ms-henglu. Once that one comment is addressed this should be good to go.

File: website/docs/r/synapse_workspace_keys.html.markdown (hunk 1 above)
Comment (ms-henglu) [reply website/docs/r/synapse_workspace_keys.html.markdown]: Thanks! I didn't notice that type. I've fixed it, please take another look.

**Review by stephybun — APPROVED** [review]
LGTM! 🎉 🙂

**Pull request merged**